packages/preact-router/src/link.tsx

```
@@ -0,0 +1,578 @@
+  const currentSearch = useRouterState({
+    select: (s) => s.location.search,
+    structuralSharing: true as any,
+  })
+
+  const from = options.from
+
+  const _options = useMemo(
+    () => {
+      return { ...options, from }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)